.github/workflows/devbuild.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       uses: actions/setup-java@v2
       with:
         distribution: 'adopt'
```

Comment:
Update this to temurin, and update the CI's setup-java, cache, checkout and upload-artifact to v3.

Reply:
Done in [0184f04](https://github.com/gnembon/chat-up/pull/22/commits/0184f04678a8a2aa0266ca042cd0916599ebf75e)